src/circuit/multicodec.ts

```diff
@@ -1,2 +1,4 @@
 
-export const RELAY_CODEC = '/libp2p/circuit/relay/0.1.0'
+export const RELAY_V1_CODEC = '/libp2p/circuit/relay/0.1.0'
```

Comment:
didn't we say we are going to remove all v1 stuff?

Reply:
Yes, all v1 code should be removed.

Reply:
We still allow connecting over v1 which is why this is still in here. Handling relay v1 has been disabled completely but the code has not been removed.

Reply:
All v1 code has been removed